waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/parse/ASTConverter.java

```diff
@@ -30,12 +30,12 @@
 import java.util.List;
 import java.util.Map;
 
+import org.apache.commons.lang3.tuple.ImmutablePair;
+import org.apache.commons.lang3.tuple.Pair;
 import org.apache.hadoop.hive.ql.parse.ASTNode;
 import org.apache.hadoop.hive.ql.parse.HiveParser;
 import org.apache.http.annotation.Experimental;
 
-import com.sun.tools.javac.util.Pair;
```

Comment:
I didn't spot this in the other PR, we should not use classes from Java tools unless we are doing something very specific with the JVM..  this is not part of the language packages